src/event-stream/event-server.ts

```diff
@@ -52,7 +52,7 @@ import {
   ClarityValueTuple,
   TxPayloadTypeID,
 } from 'stacks-encoding-native-js';
-import { ChainID } from '@stacks/transactions';
+import { ChainID, nextSignature } from '@stacks/transactions';
```

Comment:
Used import?

Reply:
I have a PR to add import sorting and removal of unused imports on save. @rafaelcr @zone117x would you be interested in that?

Reply:
Sure, that should be a separate PR without any other changes, since it will cause merge conflicts for all other pending branches.